hnn_core/gui/_viz_manager.py

```diff
@@ -334,9 +426,19 @@ def _get_ax_control(widgets, data, fig_idx, fig, ax):
     clear_button = Button(description='Clear axis')
 
     def _on_sim_data_change(new_sim_name):
-        return check_sim_plot_types(new_sim_name, plot_type_selection, data)
+        return check_sim_plot_types(
+            new_sim_name, plot_type_selection, target_data_selection, data)
+
+    def _on_target_comparison_change(new_target_name):
+        return target_comparison_change(new_target_name, simulation_selection,
+                                        data)
+
+    def _on_plot_type_change(new_plot_type):
+        return plot_type_coupled_change(new_plot_type, target_data_selection)
 
     simulation_selection.observe(_on_sim_data_change, 'value')
+    target_data_selection.observe(_on_target_comparison_change, 'value')
```

Comment:
do you need this since `target_comparison_change` just contains a `pass` ?

Reply:
You're right, I didn't notice that. Actually let me create an issue for this and a related topic we discussed last time: to separate the simulation data and recorded loaded data via flags in the data dict. That will be a quickfix